views/index.erb

```diff
@@ -0,0 +1,23 @@
+<h2>Feel at home, anywhere</h2>
+<h3>Sign up to MakersBnB</h3>
+<div class="user_info_box">
+  <form action="/user" method="post">
+    <label for="first_name">
+      First name
+      <input type="text" name="first_name">
+    </label>    
```

Comment:
I am wondering if the <input> tag should be placed outside the <label> tag. Not sure if both ways behave equally in browsers.